src/nodejs/neo-rpc/v2.9.0/client/cli/query.js

```diff
@@ -9,14 +9,17 @@
 require('module-alias/register')
 
 
-const program = require('commander')
-const _       = require('underscore')
+const program     = require('commander')
+const _           = require('underscore')
+
+const dbg         = require('nodejs_util/debug')
+const netUtil     = require('nodejs_util/network')
+const getNodesBy  = require('nodejs_neo-rpc/v2.9.0/client/module/getNodesBy')
```

Comment:
these imports are prone to breaking when updating the versions, is there any other way to import this?